src/blockchain_txn_price_oracle_v1.proto

```diff
@@ -0,0 +1,8 @@
+syntax = "proto3";
+package helium;
+
+message blockchain_txn_price_oracle_v1 {
+    bytes oracle_signature = 1;
```

Comment:
one of these is probably the public key, and the other is the signature, right?

Reply:
Yes that's right - the "oracle_signature" field is the signature _from_ the oracle validating its price estimate.

I thought the work flow would be something like:
1. Oracle estimates price
2. Oracle uses its private key to sign price
3. Price is expressed as cents
4. The price (as cents) goes in the price field
5. Signature of the price goes into "oracle_signature" field
6. The "signature" field here is supposed to be for sending the message into the blockchain?